terraform/prod-eun/README.md

```diff
@@ -0,0 +1,20 @@
+# prod-eun
+
+This directory is an experiment, using Terragrunt for DRY.
+
+It allows describing some redundant config only once (like provider config and
+remote state).
+
+It allows describing dependencies between different terraform states, as well as
+passing along values.
+
+It should also make it much easier to do a granular destruction of a "leaf"
+state.
+
+It does not use Terraform Workspaces, individual instantiations have their own
+files in this repository, making it possible to deviate or roll out changes in a
+controlled matter.
+
+In case the experiment proves successful, the goal is to slowly migrate
+everything from prod-eun (and other environments) to this approach, creating
+sibling directories.
```

Comment:
I want to add a quick start and link to some more documentation, keeping it draft for now.

Reply:
Done